src/overlays/actors/ovl_En_Elfgrp/z_en_elfgrp.c

```diff
@@ -353,10 +356,10 @@ void func_80A3A0AC(EnElfgrp* this, GlobalContext* globalCtx) {
 void func_80A3A0F4(EnElfgrp* this, GlobalContext* globalCtx) {
     if (this->unk_144 == 10) {
         play_sound(NA_SE_SY_WHITE_OUT_T);
-        if (ENELFGRP_GET(&this->actor) < ENELFGRP_4) {
+        if (STRAY_FAIRY_TYPE(&this->actor) < ENELFGRP_4) {
```

Comment:
Since this is operating on `this`, it should be `ENELFGRP_GET`:
```suggestion
        if (ENELFGRP_GET(&this->actor) < ENELFGRP_4) {
```